### ISSUE TITLE
Drag and dropping icons from file explorer don't show up in filesystem if i drag then to file manager they show up in filesystem

### ISSUE THREAD

**Issue by syziel** [godotengine/godot#78707]
### Godot version

4.0.3.stable.mono.official [5222a99f5]

### System information

Windows 10 - godot 4.0.3 stable.mono

### Issue description

Dragging and dropping icons from file explorer or desktop doesn't show up in the file system dock. If I drag them to the file manager folder directly, they show up in the file system dock.

### Steps to reproduce

Create New Project in godot then drag and drop an icon from file explorer or desktop to the file system should show the icon in the file system dock, but nothing shows. If I drag and drop from file explorer or desktop into file manager icons show in the file system.

### Minimal reproduction project

new project

**Comment by AThousandShips** [comment]
I don't think drag-drop into the editor like that is currently supported, does the documentation say it's supported?

**Comment by syziel** [comment]
I'll have to check. It even does it in 3.5.2. I have the same issue with drag and drop

**Comment by AThousandShips** [comment]
Correction it does work correctly in the editor

**Comment by syziel** [comment]
not for me i can only do it by dragging to file manager folder itself to get them to show in file system

**Comment by AThousandShips** [comment]
I can not confirm your bug, it works perfectly for me in 4.1, and don't think this would be a bug specific to 4.0.3, but unsure

**Comment by AThousandShips** [comment]
What do you mean by "show the icon in the file system dock"

What file type are you dragging?

**Comment by syziel** [comment]
yes by desktop or windows file explorer

**Comment by syziel** [comment]
when i drag them to file manager folder they appear in file system dock

**Comment by AThousandShips** [comment]
By "show the icon in the file system dock" do you mean they aren't visible there after dropping them?

**Comment by syziel** [comment]
yes. But if i put them in file manager folder they are visible in the file system dock

**Comment by syziel** [comment]
if I drag a Png icon from the desktop into Godot it is not visible, but if I put them in the file manager folder, they are visible

**Comment by AThousandShips** [comment]
You can also download the latest beta of 4.1 to see if it happens there: https://godotengine.org/article/dev-snapshot-godot-4-1-beta-3/

If it doesn't then this is a bug that is specific to 4.0.3, otherwise it's unclear what's wrong as it works perfectly on 4.1 at least on Windows 11, and might be an issue of how you are dropping the files

**Comment by syziel** [comment]
i tested in on 4.0.1 mono and everything works like it should. but doesn't work in 4.0.3.

**Comment by AThousandShips** [comment]
4.0.1 or 4.1?

**Comment by syziel** [comment]
4.0.1

**Comment by AThousandShips** [comment]
Okay, thank you!

**Comment by AThousandShips** [comment]
Did you reach any solution for this? Did you mean to close?

**Comment by syziel** [comment]
no sill trying

**Comment by AThousandShips** [comment]
It'd be more helpful if you could provide screenshots or video of the bug so we can help figuring out what is going on

**Comment by syziel** [comment]
how would i make a video?

**Comment by syziel** [comment]
best i could do

![Animation](https://github.com/godotengine/godot/assets/117110946/518289b3-80e7-42d6-a4c2-7f032928438c)

**Comment by syziel** [comment]
Best I could do

![Animation](https://github.com/godotengine/godot/assets/117110946/3391c573-39a9-4100-90fd-1651eda158e8)

**Comment by AThousandShips** [comment]
Thank you that shows it clearly!

**Comment by Jeyanthinath** [comment]
Any update on this, I ran into this issue recently in `v4.3.stable.official [77dcf97d8]`.

This makes the usability really bad; I need open a separate window to manage the assets. 

Also, I need to search through whole bunch of online forums to land this place, otherwise I thought I messed up my folder structure.

**Comment by D-MAS** [comment]
got this on 4.4

**Comment by davidkovacwebdev** [comment]
Got this on 4.4 as well on Mac

I've dragged and dropped an image from the Finder to res:// sprites.
This resulted in entire godot project somehow being duplicated within the sprites folder. 

When I removed all the extra files and restarted godot the file explorer works as expected

**Comment by BastionNtB** [comment]
Getting this issue in v4.6.2.stable.official [71f334935] on Linux. Dragging files into FileSystem from Dolphin does not update the list of files in FileSystem, but if you restart Godot, it shows they have been added.
